### ISSUE TITLE
Review of text on stats module

### ISSUE THREAD

**Issue by chrisb83** [scipy/scipy-articles#97]
I reviewed the text on `stats`:

I agree with @tylerjereddy (see #65 
) that the text in paper.tex could be shortened. From my point of view, the following are the main points if we speak about distributions in `stats`:

- more than 100 distributions available (~80 continuous, 12 discrete and 10 multivariate)
- consistent framework that implements methods to sample rvs, evaluate the pdf / cdf and to fit parameters for every distribution
- generally relies on specific implementations for each distributions, otherwise defaults to generic methods (such as deriving the cdf from the pdf via numerical integration)

Questions / ideas to shorten the current text:

- comment on random_state: is this needed? I assume most users expect that they can provide a seed.
- multinomial: same as random_state, probably expected that this exists. note: "only multivariate" contradicts statement that there are 12 multivariate
- rv_histogram: do we want to sell this as an important feature? advantage / use case probably not clear from the text

When mentioning other features in subpackages.tex, the scope is quite narrow:

> statistical tests, including Pearson's correlation, Spearman's rank-order correlations, Kendall's tau, chi-squared test and its generalization as the Cressie-Read power divergence, contingency table tests including Fisher's exact test and Mood's median test, and many more; and assorted transformations and statisticsof data

- we could mention more generic test classes such as testing correlation, equality of the mean / median / variance, whether a sample follows a certain distribution etc and give a few well-known examples. (the ones given now sucha s Mood and Cressie-Read are not very well-known, just my opinion)
- what about KDE?
- statistical distances?

Happy to get your view.

**Comment by tylerjereddy** [comment]
You may not get too much feedback -- the subset of devs working on the paper actively is pretty small, but I trust your judgement far more than mine on `stats` content, and I probably ended up drafting some of the stats material, unfortunately!

@mdhaber has been good about reviewing PRs carefully, so I think if you clean up the stats stuff a little he'd be happy to review.

**Comment by mdhaber** [comment]
I would be happy to review. I'm also starting to dig into my own edits now that we've gone through a fair portion of Tyler's list. I haven't read the most recent version of the stats section, but when I do I can let you know what I think about it. If you want to submit a PR before then, though, that also works.

**Comment by chrisb83** [comment]
> If you want to submit a PR before then, though, that also works.

I would be happy to get your review of the current version to see if you agree with my comments above before I create a PR.

**Comment by tylerjereddy** [comment]
@chrisb83 There's a question about stats content in #113 -- can you take a look -- you can ignore the bunch of other stuff changed there

**Comment by rgommers** [comment]
```
Questions / ideas to shorten the current text:

    comment on random_state: is this needed? I assume most users expect that they can provide a seed.
    multinomial: same as random_state, probably expected that this exists. note: "only multivariate" contradicts statement that there are 12 multivariate
    rv_histogram: do we want to sell this as an important feature? advantage / use case probably not clear from the text
```
agreed, those are all not very important, could be shortened.

> From my point of view, the following are the main points if we speak about distributions in stats:

agreed as well

```

    we could mention more generic test classes such as testing correlation, equality of the mean / median / variance, whether a sample follows a certain distribution etc and give a few well-known examples. (the ones given now sucha s Mood and Cressie-Read are not very well-known, just my opinion)
    what about KDE?
    statistical distances?
```
Correlation/hypothesis testing and KDE make sense. Statistical distances is minor perhaps, there's only a couple (especially in 1.0 IIRC, we recently added 1 or 2). Could leave that one out I think.

**Comment by chrisb83** [comment]
@rgommers thanks for your feedback, i will try to submit a PR in the next days

do we only Mention functionality present in SciPy 1.0 or can I also mention more recent features added in 1.10 or 1.20?

**Comment by mdhaber** [comment]
@chrisb83 Present in SciPy 1.0, please

**Comment by rgommers** [comment]
only SciPy 1.0, that's where we drew the line - there will likely be some other paper in the future about newer additions

**Comment by mdhaber** [comment]
@chrisb83 Is this closed by #125?

**Comment by chrisb83** [comment]
Yes
